### PR TITLE
fix broken yast module startup (bsc#1122473)

### PR DESCRIPTION
**Pull request by wfeldt**
For some reason the constructed command line from the ncurses control center
looks like this:
> y2start bootloader --arg ncurses -name YaST2 -icon yast

with an superfluous '--arg'. I've no idea where that comes from but
the check in this patch is a safe-guard against this anyway.

### CODE REVIEW

**Comment by mvidner** [comment]
This looks like a duplicate of https://github.com/yast/yast-yast2/pull/887

File: scripts/yast2 [hunk 1]
```diff
@@ -295,8 +295,10 @@ if [ -n "$CMDLINE_HELP" ]; then
 fi
 ARGS=( )
 for i in "$@"; do
-  ARGS+=( "--arg" )
-  ARGS+=( "$i" )
+  if [ -n "$i" ] ; then
```
Comment (jreidinger) [inline scripts/yast2]: as @mvidner already pointed out it should be fixed by different patch. This will explicitly filter all intentionally empty parameters, which is quite big change. so :-1: for me

**Comment by wfeldt** [comment]
not needed
